src/Snap/Snaplet/#SqliteSimple.hs#

```diff
@@ -0,0 +1,227 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
ahh, I see that you removed this in a later commit.  Maybe use interactive rebase to clean up accidental add + delete from history?